app/modules/detectx/provides.php

```diff
@@ -0,0 +1,14 @@
+<?php
+//
+return array(
+    'client_tabs' => array(
+        'detectx-tab' => array('view' => 'detectx_tab', 'i18n' => 'detectx.clienttitle', 'badge' => 'detectx-cnt'),
+    ),
+    'listings' => array(
+       'detectx' => array('view' => 'detectx_listing', 'i18n' => 'detectx.clienttitle'),
```

Comment:
detectx.clienttitle does not exist in the locale.json. You may want to change that to detectx.title (this controls the name in the dropdown in the listing menu)